### PR TITLE
Add alwaysShowTooltip to set ToolTip of Slider show always

### PR DESCRIPTION
**Pull request by orzyyyy**
When alwaysShowTooltip is true, Tooltip will show always.

[#12743](https://github.com/ant-design/ant-design/issues/12743)

### CODE REVIEW

File: components/slider/demo/show-tooltip.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  en-US: Control visible of ToolTip
+---
+ ## zh-CN
+ 当 `alwaysShowTooltip` 为 `true` 时，将始终显示ToolTip。
+ ## en-US
+ When `alwaysShowTooltip` is `true`, ToolTip will show always.
+ ````jsx
+import Slider from '..';
+ ReactDOM.render(<Slider defaultValue={30} alwaysShowTooltip />, mountNode);
+````
```
Comment (afc163) [inline components/slider/demo/show-tooltip.md]: 保留合适的空行，可以参考其他 demo。

**Comment by afc163** [comment]
没看到 alwaysShowTooltip 的代码？

File: components/slider/demo/show-tooltip.md (hunk 1 above)
Comment (zy410419243) [reply components/slider/demo/show-tooltip.md]: 已改

**Comment by afc163** [comment]
检查下 test case。

**Comment by zy410419243** [comment]
show-tooltip.md里得注意下，别的demo里都是
`import { Slider } from 'antd'`
合并到主分支的时候可能要改一下这个。这里不这么写就没法通过ci

**Comment by afc163** [comment]
没道理，理论上没区别的。

**Comment by ilanus** [comment]
maybe better `open` or `visible` instead of `alwaysShowTooltip`.

**Comment by zy410419243** [comment]
@ilanus if `open`, I don't know what is opening, the same as `visible`, `alwaysShowTooltip` can be easier to know what I am doing.

**Comment by afc163** [comment]
Maybe `tooltipVisible`?

**Comment by afc163** [comment]
We can make `tooltipVisible` as a controlled value, which means:

```jsx
<Slider tooltipVisible />  // always show tooltip
<Slider tooltipVisible={false} />  // never show tooltip
```

**Comment by afc163** [comment]
@zy410419243 Yes, like `visible` of `Tooltip`.

**Comment by ztplz** [comment]
Just a little comment.  `tooltipVisible` is better than `alwaysShowTooltip` IMO.  it's necessary to keep `props name` as simple as possible.

**Comment by zy410419243** [comment]
Something went wrong, I will create a new PR later.

**Pull request closed without merging**